### PR TITLE
[Fix]: Fix outdated problem

### PR DESCRIPTION
**Pull request by YuanLiuuuuuu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/zh_cn/algorithms/mae.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+| :------: | :-------------: | :---------------: | :-------------------------------------------------: | :---------------------------------------------------------------------------------------: | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| ViT-B/16 |       400       |       83.1        | [config](./mae_vit-b-p16_8xb512-coslr-400e_in1k.py) | [config](../../benchmarks/classification/imagenet/vit-b-p16_ft-8xb128-coslr-100e_in1k.py) | [model](https://download.openmmlab.com/mmselfsup/mae/mae_vit-base-p16_8xb512-coslr-400e_in1k-224_20220223-85be947b.pth) &#124; [log](https://download.openmmlab.com/mmselfsup/mae/mae_vit-base-p16_8xb512-coslr-300e_in1k-224_20220210_140925.log.json) |
```
Comment (fangyixiao18) [inline docs/zh_cn/algorithms/mae.md]: Use github link instead of relative path to prevent the link error.

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]

File: docs/en/algorithms/mae.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+| :------: | :-------------: | :---------------: | :-------------------------------------------------: | :---------------------------------------------------------------------------------------: | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| ViT-B/16 |       400       |       83.1        | [config](./mae_vit-b-p16_8xb512-coslr-400e_in1k.py) | [config](../../benchmarks/classification/imagenet/vit-b-p16_ft-8xb128-coslr-100e_in1k.py) | [model](https://download.openmmlab.com/mmselfsup/mae/mae_vit-base-p16_8xb512-coslr-400e_in1k-224_20220223-85be947b.pth) &#124; [log](https://download.openmmlab.com/mmselfsup/mae/mae_vit-base-p16_8xb512-coslr-300e_in1k-224_20220210_140925.log.json) |
+
+
+## Citation
+
+```bibtex
+@article{He2021MaskedAA,
```
Comment (Jiahao000) [inline docs/en/algorithms/mae.md:46]: Please make the bibtex format consistent.

File: docs/en/algorithms/mocov3.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+The **Linear Evaluation** result is obtained by training a linear head upon the pre-trained backbone. Please refer to [vit-small-p16_8xb128-coslr-90e_in1k](https://github.com/open-mmlab/mmselfsup/blob/master/configs/benchmarks/classification/imagenet/vit-small-p16_8xb128-coslr-90e_in1k.py) for details of config.
+
+| Self-Supervised Config                                                                                                                                                                | Linear Evaluation |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| [vit-small-p16_32xb128-fp16-coslr-300e_in1k-224](https://github.com/open-mmlab/mmselfsup/blob/master/configs/selfsup/mocov3/mocov3_vit-small-p16_32xb128-fp16-coslr-300e_in1k-224.py) | 73.19             |
+
+## Citation
+
+```bibtex
+@Article{chen2021mocov3,
```
Comment (Jiahao000) [inline docs/en/algorithms/mocov3.md]: Please update the bibtex as mocov3 has been accepted by ICCV.

**Review by Jiahao000 — CHANGES_REQUESTED** [review]

File: configs/selfsup/mocov3/README.md [hunk 4]
```diff
@@ -33,10 +33,12 @@ The **Linear Evaluation** result is obtained by training a linear head upon the
...
-  title   = {An Empirical Study of Training Self-Supervised Vision Transformers},
-  journal = {arXiv preprint arXiv:2104.02057},
-  year    = {2021},
+@InProceedings{Chen_2021_ICCV,
+    author    = {Chen, Xinlei and Xie, Saining and He, Kaiming},
+    title     = {An Empirical Study of Training Self-Supervised Vision Transformers},
+    booktitle = {Proceedings of the IEEE/CVF International Conference on Computer Vision (ICCV)},
+    month     = {October},
+    year      = {2021},
+    pages     = {9640-9649}
```
Comment (Jiahao000) [inline configs/selfsup/mocov3/README.md]: No need to add `month` and `pages`. For the format issue, please refer to the format of previous algorithms.

**Review by Jiahao000 — CHANGES_REQUESTED** [review]

**Review by fangyixiao18 — APPROVED** [review]

**Pull request merged**